css-module-bikeshed/Overview.bs

```diff
@@ -159,3 +159,7 @@ Internal display model: the 'foo' property {#the-foo-property}
 			DOMString baz = "qux";
 		};
 	</pre>
+
+<h2 class=no-num id=priv-sec>Privacy and Security Considerations</h2>
+
+Make some considerations about privacy and security.
```

Comment:
I don't have a strong opinion about that, but there is no other issue marking a 'must edit'.

Sebastian